components/src/hardware-sim/Pipette/PipetteRender.tsx

```diff
@@ -36,19 +38,31 @@ export const PipetteRender = (props: PipetteRenderProps): JSX.Element => {
   const x = labwareDef.wells.A1.x - cx
   const y = channels === 1 ? labwareDef.wells.A1.y - cy : MULTI_CHANNEL_Y_OFFSET
 
+  let boxWidth
+  let probeOffsetX = 0
+  let probeOffsetY = 0
```

Comment:
nit, can we type these?

```suggestion
  let boxWidth: number
  let probeOffsetX: number = 0
  let probeOffsetY: number = 0
```